code/modules/bluespace_exploration/encounter/templates_syndicate.dm

```diff
@@ -0,0 +1,11 @@
+/datum/map_template/shuttle/ship/syndicate
+	faction = /datum/faction/syndicate
+
+/datum/map_template/shuttle/ship/syndicate/fighter
+	name = "Syndicate Fighter"
+	id = "syndie_fighter"
+	suffix = "syndicate_fighter"
+	port_id = "encounter"
+
+	difficulty = 3	//These are super easy and super small
+	allowed_weapons = list()
```

Comment:
🤔

Reply:
Intent is to have the list defined in the datum, so it can't be created in new()